core/src/main/kotlin/org/evomaster/core/search/Individual.kt

```diff
@@ -2,21 +2,32 @@ package org.evomaster.core.search
 
 import org.evomaster.core.search.gene.Gene
 import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.tracer.TraceableElement
+import org.evomaster.core.search.service.tracer.TrackOperator
 
 /**
  * An individual for the search.
  * Also called "chromosome" in evolutionary algorithms.
  * In our context, most of the time an Individual will represent
  * a single test case, composed by 1 or more "actions" (eg, calls
  * to a RESTful API, SQL operations on a database or WireMock setup)
+ *
+ * Individual allows to track its evolution, created by sampler, changed by mutator or crossover
+ * @property trackOperator presents an operatorTag to change an individual, e.g., mutator
+ * @property track is a list of Individual, indicating its evolution
  */
-abstract class Individual {
+abstract class Individual : TraceableElement{
+
+    constructor(trackOperator: TrackOperator? = null, traces : MutableList<out Individual>? = null) : super(trackOperator, traces)
 
     /**
-     * Make a deep copy of this individual
+     * Make a deep next of this individual
```

Comment:
when you refactored the variable name from "copy" to "next", didn't you by mistake also did in the comments without double-checking it?

Reply:
yes. it is a mistake.